lib/widgets/theme.dart

```diff
@@ -182,6 +184,8 @@ class DesignVariables extends ThemeExtension<DesignVariables> {
       btnLabelAttLowIntDanger: const Color(0xffff8b7c),
       btnLabelAttMediumIntDanger: const Color(0xffff8b7c),
       composeBoxBg: const Color(0xff0f0f0f),
+      composeBoxBorderBar: Colors.white.withValues(alpha: 0.1),
+      composeBoxShadow: const Color(0xff242424),
```

Comment:
The Figma uses an existing variable for the shadow color, `bg-top-bar`; we should do the same.